docs/modules/ROOT/pages/pallets/proxy.adoc

```diff
@@ -0,0 +1,357 @@
+:source-highlighter: highlight.js
+:highlightjs-languages: rust
+:github-icon: pass:[<svg class="icon"><use href="#github-icon"/></svg>]
+
+= pallet_proxy link:https://github.com/paritytech/polkadot-sdk/tree/release-polkadot-v1.5.0/substrate/frame/proxy[{github-icon},role=heading-link]
+
+Branch/Release: `release-polkadot-v1.5.0`
+
+== Purpose
+
+This pallet gives an ability to delegate the rights to execute some certain calls types from one origin to another. 
+
+== Glossary
+
+* _Announcement_ — a statement of call hash for the proxy to execute in some future block. Required for some proxies where delay is specified
+* _Delay_ — number of block that should pass from announcement before call execution
+* _Delegatee_ — account that was granted call execution rights with proxy creation
+* _Delegator_ — account that granted call execution rights with proxy creation
+* _Proxy_ — statement of call execution rights transfer from delegator to delegatee. Specified by proxy type and delay.
+* _Proxy type_ — type of calls that can be executed using this proxy.
+* _Pure account_ — accounted that was spawned only to be a delegatee for some proxy.
+
+== Config
+
+* Pallet-specific configs
+** `ProxyType` -- a type that describes different variants of proxy. It must implement `Default` trait and `InstanceFilter<RuntimeCall>` trait.
+** `ProxyDepositBase` -- a base amount of currency that defines a deposit for proxy creation.
+** `ProxyDepositFactor` -- an amount of currency that will be frozen along with the `ProxyDepositBase` for each additional proxy.
+** `MaxProxies` -- maximum number of proxies that single account can create.
+** `MaxPending` -- maximum number of announcements that can be made per account.
+** `AnnouncementDepositBase` -- a base amount of currency that defines a deposit for announcement creation.
+** `AnnouncementDepositFactor` -- an amount of currency that will be frozen along with the `AnnouncementDepositBase` for each additional announcement.
+* Commong configs:
```

Comment:
I really like the distinction on common configs and pallet specific ones. 

Should we have this in the pallet template as well?

Reply:
Good idea!
Added it.